File: src/main/java/sh/miles/cosmictools/util/CurrentDirectory.java
package sh.miles.cosmictools.util;

import java.nio.file.Path;

public final class CurrentDirectory {

    private static Path CWD = Path.of(".");

    public static Path cwd() {
        return CWD;
    }

    public static void cwd(Path path) {
        CWD = path;
    }

}

File: src/main/kotlin/sh/miles/cosmictools/utils/CurrentDirectory.kt
package sh.miles.cosmictools.utils

import java.nio.file.Path

object CurrentDirectory {

    private var CWD: Path = Path.of(".")

    fun cwd(): Path {
        return CWD
    }

    fun cwd(path: Path) {
        CWD = path
    }
}
